src/libos/crates/async-io/src/event/poller.rs

```diff
@@ -84,6 +84,21 @@ impl Pollee {
         revents
     }
 
+    pub fn fast_poll(
```

Comment:
Do not get the point. Why don't you use poll (mask, none) in the code directly?

Reply:
The intention of adding `fast_poll` is good. But the API does not feel good to me.

**The first problem is that the API is NOT easy-to-use.** The user has to pass in a `first_time` variable. And the user must not accidentally modify it. If we have `fast_poll` method, I hope it can be more automatic and transparent to users, hiding the decision condition (which is `first_time`) from the user.

**The second problem is that the API is prone to misuse.** After the first call, `fast_poll` switches to `poll`, assuming that the first call must have connected the poller and pollee. But this assumption is not true: _the `poll` method connects a poller and a pollee only when the returned events are empty_. In most use cases, if the user code finds the return events are non-empty, then it will stop polling; so `fast_poll` is ok. But if, for some reasons, the user code decides to poll again even when the non-empty events are returned in the last time, then using `fast_poll` may lead to wait forever.

Reply:
@guzongmin @tatetian Thank you for your review. I try a different approach to separate the `connect` and `poll` semantics for users to call based on their needs.